<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgEntry</class>
 <widget class="QDialog" name="CDlgEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>389</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>445</width>
    <height>389</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>445</width>
    <height>389</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_0">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>52</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Class</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cExtra2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>260</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>goal</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>52</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Sprite</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cF40">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>235</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>151</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>displayed in</string>
        </property>
        <widget class="QCheckBox" name="cD01">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>30</y>
           <width>82</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>normal</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cD04">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>90</y>
           <width>82</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>hell</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cD02">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>60</y>
           <width>82</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>nightmare</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cD08">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>120</y>
           <width>82</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>insane</string>
         </property>
        </widget>
       </widget>
       <widget class="QToolButton" name="btnPrevImage">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>90</y>
          <width>25</width>
          <height>22</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Previous Image</string>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
       <widget class="QToolButton" name="btnObject">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>140</y>
          <width>91</width>
          <height>24</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Displays the object property page</string>
        </property>
        <property name="text">
         <string>Edit Sprite</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbTriggerKey">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="cF20">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>260</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>95</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Trigger Key</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cHidden">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>210</y>
          <width>82</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>hidden</string>
        </property>
       </widget>
       <widget class="QToolButton" name="btnNextImage">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>114</y>
          <width>25</width>
          <height>22</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Next Image</string>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cF80">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>210</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>190</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Attributes</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>190</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Trigger flip</string>
        </property>
       </widget>
       <widget class="QLabel" name="sClass">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>201</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="sObject">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>23</y>
          <width>201</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="sImage">
        <property name="geometry">
         <rect>
          <x>311</x>
          <y>3</y>
          <width>111</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="cFrozen">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>235</y>
          <width>82</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>frozen</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Hints</string>
       </attribute>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Hints / Description</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="eHint">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>351</width>
          <height>161</height>
         </rect>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>224</y>
          <width>371</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>You can access this string at runtime through scripting using Sprite:getString(). See documentation for more details.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Path</string>
       </attribute>
       <widget class="QLabel" name="sPathSize">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Path Size:</string>
        </property>
       </widget>
       <widget class="QLabel" name="sPath">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDeletePath">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>20</y>
          <width>93</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete Path</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cPlayback">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>131</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Playback path</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cLoop">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Loop</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cIgnoreGravity">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>110</y>
          <width>111</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Ignore gravity</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cIgnoreSolid">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>140</y>
          <width>111</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Ignore solid</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cAttackPlayer">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>111</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Attack Player</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cCustomHandler">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>204</y>
          <width>121</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Custom Handler</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cReturnOrigin">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>170</y>
          <width>131</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Return to origin</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbCustomHandler">
        <property name="geometry">
         <rect>
          <x>171</x>
          <y>200</y>
          <width>101</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="cApplyActive">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>170</y>
          <width>161</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Only apply when active</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>237</y>
          <width>341</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>You have to create a custom path for this sprite before you can automatically play it back.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnRawPath">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>50</y>
          <width>93</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Raw Path</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbTriggerKey</tabstop>
  <tabstop>btnPrevImage</tabstop>
  <tabstop>btnNextImage</tabstop>
  <tabstop>btnObject</tabstop>
  <tabstop>cD01</tabstop>
  <tabstop>cD02</tabstop>
  <tabstop>cD04</tabstop>
  <tabstop>cD08</tabstop>
  <tabstop>cHidden</tabstop>
  <tabstop>cFrozen</tabstop>
  <tabstop>cExtra2</tabstop>
  <tabstop>cF80</tabstop>
  <tabstop>cF40</tabstop>
  <tabstop>cF20</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>btnDeletePath</tabstop>
  <tabstop>eHint</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDlgEntry</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDlgEntry</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
